apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/DataSetManifest.java

```diff
@@ -182,23 +183,6 @@ public String getIncomingS3Key() {
     return getId().computeS3Key(manifestKeyIncomingLocation);
   }
 
-  /**
-   * Gets the timestamp text.
-   *
-   * @return the {@link String} representation of {@link #getTimestamp()} used to identify this
-   *     {@link DataSetManifest} in S3 and elsewhere
-   */
-  public String getTimestampText() {
-    /*
-     * Design note: As discovered in CBBD-298, Java's DateTimeFormatter and
-     * Instant classes don't always preserve the precision of parsed
-     * timestamps. Accordingly, we need to store this value as a String to
-     * ensure that the S3 key of the manifest isn't mangled.
-     */
```

Comment:
Hate to lose this information.  This comment should be moved to the field.

Reply:
I moved this comment to the field declaration